### DISCUSSION TITLE
NFT

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/security-and-auditing-full-course-s23#27] [Q&A]
Can anyone please help me how to solve nft challenge here
<img width="640" alt="image" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/85685981/954ccc9e-1092-4280-b9bb-c4fe14305070">
<img width="960" alt="image" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/85685981/43f832a2-ab7f-40fe-bcce-38481c1afca5">

**Comment by cromewar** [comment]
Hello @panditdhamdhere 
The challenges are not supposed to be spoiled, so take a double check and read the contract, there you have the the execute this function comment, read the input parameters and try to solve it 😉
